README.md

```diff
@@ -25,6 +38,7 @@ sudo rosdep init
 rosdep update
 echo "source /opt/ros/indigo/setup.bash" >> ~/.bashrc
 source ~/.bashrc
+sudo apt-get install ros-indigo-rosjava-build-tools
```

Comment:
The app is compatible with both indigo and kinetic. Would be nice to abstract this as <ros-distro> or just make a note about this.